### PR TITLE
rev ES and allow anonymous access to kibana

### PR DESCRIPTION
**Pull request by coffeepac**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

bumps elasticsearch and kibana to latest oss versions
adds rolebinding to allow anonymous access to kibana dashboard.
  - this isn't intended to be a production-quality configuration, we should let users be able to get into their data immediately.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by monotek — APPROVED** [review]
/lgtm
/approve

**Comment by coffeepac** [comment]
@monotek this never got merged for some reason.  I've fixed the rebase conflict.  lemme know if you want to merge this or close and re-open against the future location.

**Review by monotek — APPROVED** [review]

**Comment by monotek** [comment]
@coffeepac 

Lets merge it here.
I've added a link to my pr in https://github.com/kubernetes-sigs/instrumentation-addons/pull/1 to pickup the changes later :)

File: cluster/addons/fluentd-elasticsearch/kibana-deployment.yaml [hunk 1]
```diff
@@ -53,3 +53,28 @@ spec:
...
+kind: ClusterRoleBinding
+metadata:
+  name: kubernetes-dashboard-anonymous
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kubernetes-dashboard-anonymous
+subjects:
+- kind: User
+  name: system:anonymous
```
Comment (liggitt) [inline cluster/addons/fluentd-elasticsearch/kibana-deployment.yaml:80]: permission grants to anonymous users are *not* expected in these addons. revert opened in https://github.com/kubernetes/kubernetes/pull/102010

**Pull request merged**